test/e2e/util_test.go

```diff
@@ -598,31 +599,117 @@ func updateIngressConfigSpecWithRetryOnConflict(t *testing.T, name types.Namespa
 	})
 }
 
-// updateInfrastructureConfigSpecWithRetryOnConflict gets a fresh copy
+// updateInfrastructureConfigWithRetryOnConflict gets a fresh copy
 // of the named infrastructure object, calls updateSpecFn() where
 // callers can modify fields of the spec, and then updates the infrastructure
 // config object. If there is a conflict error on update then the
 // complete operation of get, mutate, and update is retried until
 // timeout is reached.
-func updateInfrastructureConfigSpecWithRetryOnConflict(t *testing.T, name types.NamespacedName, timeout time.Duration, mutateSpecFn func(*configv1.InfrastructureSpec)) error {
+// After updating, it will check if the desired spec.Endpoints where applied to
+// the infraconfig
+func updateInfrastructureConfigWithRetryOnConflict(t *testing.T, name types.NamespacedName, timeout time.Duration, mutateSpecFn func(*configv1.InfrastructureSpec)) error {
```

Comment:
The function doesn't retry "OnConflict" anymore, plus it does "update + check" type of action.  It has to be renamed.

Reply:
changed the name to `updateAndVerifyInfrastructureConfigWithRetry`